internal/state/store.go

```diff
@@ -100,13 +109,17 @@ type Store interface {
 // dbStore wraps a db (github.com/tendermint/tm-db)
 type dbStore struct {
 	db dbm.DB
+
+	// discardABCIResponses determines if all
+	// ABCI responses are persisted or just the last
+	discardABCIResponses bool
 }
 
 var _ Store = (*dbStore)(nil)
 
 // NewStore creates the dbStore of the state pkg.
```

Comment:
Godoc comment should be expanded here to detail `discardABCIResponses`, because the default public rendering will not show the underlying `dbStore` type, nor would it show the unexported `discardABCIResponses` field on it.